### PR TITLE
[FIX] locustio has moved to locust

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: examples/locustfile.py [hunk 1]
```diff
@@ -9,32 +9,28 @@
 
 
 class BackendReadOnlyUser(OdooRPCLocust):
-    min_wait = 500
-    max_wait = 5000
+    wait_time = between(500, 5000)
     weight = 10
 
-    task_set = BackendReadOnlyBehavior
+    tasks = {BackendReadOnlyBehavior}
```
Comment (sebalix) [inline examples/locustfile.py]: Hi,
Noob question: why you are using `{}` here and `[]` on the other places?

Comment (max3903) [reply examples/locustfile.py]: @sebalix Because both options are available: https://docs.locust.io/en/stable/writing-a-locustfile.html#id1

Comment (sebalix) [reply examples/locustfile.py]: The doc says that `tasks` could be a list or a dictionary (to bind int ratio), but here you are simply declaring a `set` I think, so it'll act like a `list` by Locust, but it's a bit misleading?

Comment (max3903) [reply examples/locustfile.py]: Fixed.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**